mantidproject/mantid
# ContainerSubtraction scaleX for off centre peaks

Fixes #13993

An additional Container shift option has been added to the interface to allow for containers that do not match the peak of the sample to be shifted into position.
# To Test
- Open ContainerSubtraction (Interfaces > Indirect > Corrections > ContainerSubtraction)
- irs26176_graphite002_red.nxs for the sample
- irs26174_graphite002_red.nxs for the container
  - Please get these files from `\\olympic\babylon5\Public\ElliotOram\Corrections\ContainerSubtraction\-0.2 shifted can` as the can has already been shifted -0.2 in x for testing
- Check the container (irs26174) is off zero by -0.2 
  - plot one of the spectrum and check the peak centre = -0.2
- Check the `Shift x-values of container by adding` option and set the shift to `+0.20`
- Click Run, you may be prompted with a warning saying the binning does not match and asking if you would like to rebin, click `yes`
  Ensure that the plot now show the aligned Container and Sample with the Subtraction and it looks correct.
